broker/cloud_run/lsst/classify_upsilon/main.py

```diff
@@ -98,7 +98,7 @@ def _classify_with_upsilon(alert_lite_df: pd.DataFrame) -> dict:
         # ---Extract data
         filter_diaSources = alert_lite_df[alert_lite_df["band"] == band]
         flux_gt_zero = filter_diaSources["psfFlux"].to_numpy() > 0
-        upsilon_dict[f"n_data_points_{band}_band"] = flux_gt_zero.sum()
+        upsilon_dict[f"n_data_points_{band}_band"] = int(flux_gt_zero.sum())
```

Comment:
```suggestion
        upsilon_dict[f"n_data_points_{band}_band"] = flux_gt_zero.sum().item()
```

`.item()` is the numpy method to convert to standard python type and may be better at handling edge case. I'm not sure whether it's really a big deal here but seems safer in case.